build/namespace_test.stub.php

```diff
@@ -0,0 +1,75 @@
+<?php
```

Comment:
The place to add these kinds of tests would be https://github.com/php/php-src/blob/master/ext/zend_test/test.stub.php. This should also show us the generated code and the fact that the result builds.